com.unity.ml-agents/Editor/BehaviorParametersEditor.cs

```diff
@@ -98,6 +101,19 @@ void DisplayFailedModelChecks()
             {
                 sensorComponents = behaviorParameters.GetComponents<SensorComponent>();
             }
+
+            int observableSensorSizes = 0;
```

Comment:
Observable seems to be the name for the members marked with the Observable attribute. Maybe in this code it should have a mode descriptive name. `ReflexionObservable` ?

Reply:
Do you mean specifically in this part of the code, or in general? We already have "Reflection" in the namespace for the attribute; I don't think we need it in the attribute name as well.

Reply:
I mean the name of this variable specifically.

Reply:
Renamed to `observableAttributeSensorTotalSize`

Reply:
(and added a few comments)